shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -0,0 +1,234 @@
+    mIndentationLevel++;
+    for (Map.Entry<String, Long> totalBytesTier : sumTotalBytesOnTiersMap.entrySet()) {
+      long value = totalBytesTier.getValue();
+      if (value != 0) {
+        print("Tier: " + totalBytesTier.getKey()
+            + "  Size: " + FormatUtils.getSizeFromBytes(value));
+      }
+    }
+
+    mIndentationLevel--;
+    print("Used Capacity: "
+        + FormatUtils.getSizeFromBytes(sumUsedBytes));
+    mIndentationLevel++;
+    for (Map.Entry<String, Long> usedBytesTier : sumUsedBytesOnTiersMap.entrySet()) {
+      long value = usedBytesTier.getValue();
+      if (value != 0) {
+        print("Tier: " + usedBytesTier.getKey()
+            + "  Size: " + FormatUtils.getSizeFromBytes(value));
+      }
+    }
+
+    mIndentationLevel--;
+    if (sumCapacityBytes != 0) {
+      int usedPercentage = (int) (100L * sumUsedBytes / sumCapacityBytes);
+      print(String.format("Used Percentage: " + "%s%%", usedPercentage));
+      print(String.format("Free Percentage: " + "%s%%", 100 - usedPercentage));
+    }
+    print("");
+
+    // Print cached worker information
+    mPrintStream.println(mStringBuilder.toString());
+  }
+
+  /**
+   * Gets the report worker options.
+   *
+   * @param cl CommandLine that contains the client options
+   * @return ReportWorkerOptions to get report worker information
+   */
+  private ReportWorkerOptions getOptions(CommandLine cl) throws IOException {
+    ReportWorkerOptions options = ReportWorkerOptions.defaults();
+    options.setFieldRange(new HashSet<>(Arrays.asList(ReportWorkerInfoField.ADDRESS,
+        ReportWorkerInfoField.CAPACITY_BYTES, ReportWorkerInfoField.CAPACITY_BYTES_ON_TIERS,
+        ReportWorkerInfoField.ID, ReportWorkerInfoField.LAST_CONTACT_SEC,
+        ReportWorkerInfoField.START_TIME_MS, ReportWorkerInfoField.USED_BYTES,
+        ReportWorkerInfoField.USED_BYTES_ON_TIERS)));
+    if (cl.hasOption("live")) {
+      options.setWorkerRange(WorkerRange.LIVE);
+    } else if (cl.hasOption("lost")) {
+      options.setWorkerRange(WorkerRange.LOST);
+    } else if (cl.hasOption("worker")) {
+      options.setWorkerRange(WorkerRange.SPECIFIED);
+      String addressString = cl.getOptionValue("worker");
+      String[] addressArray = addressString.split(",");
+      options.setAddresses(new HashSet<>(Arrays.asList(addressArray)));
+    }
+    return options;
+  }
+
+  /**
+   * @return initialized tier map
+   */
+  private Map<String, Long> initializeTierMap() {
+    Map<String, Long> map = new HashMap<>();
+    map.put("MEM", 0L);
+    map.put("SSD", 0L);
+    map.put("HDD", 0L);
+    return map;
+  }
+
+  /**
+   * Prints indented information.
+   *
+   * @param text information to print
+   */
+  private void print(String text) {
+    String indent = Strings.repeat(" ", mIndentationLevel * INDENT_SIZE);
+    mPrintStream.println(indent + text);
+  }
+
+  /**
+   * @return capacity command usage
+   */
+  public static String getUsage() {
```

Comment:
is there an advantage to making this method over referencing `USAGE` directly?

Reply:
Deleted the private final USAGE string and getUsage() command will directly return the String Usage.